app/controllers/organizations/staff/external_form_submission_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Organizations::Staff::ExternalFormSubmissionController < Organizations::BaseController
+  layout "dashboard"
+  def index
+    authorize! StaffAccount, context: {organization: Current.organization}
```

Comment:
Then I would need to create an `ExternalFormUpload` model, table, and so on. Because when creating its policy I'm running into errors when authorizing the index method. ? 🤔

Reply:
Hmm we should not need a new modal. Check out the `DashboardPolicy` or the Invitation policies - these act on resources that are different from the name of the policy. 

For example, in https://github.com/rubyforgood/pet-rescue/blob/main/app/controllers/organizations/staff/dashboard_controller.rb we pass the org as the context and then in the policy https://github.com/rubyforgood/pet-rescue/blob/main/app/policies/organizations/dashboard_policy.rb we check for the persence of the permission

Reply:
Thank you, with that I managed to understand the code more and found out where I was mistaken/misunderstanding. 😄

Reply:
Good stuff!